txt-suite does not build

```
$ go get -u github.com/9elements/converged-security-suite/cmd/txt-suite
# github.com/9elements/converged-security-suite/pkg/hwapi
../../9elements/converged-security-suite/pkg/hwapi/cpuid_amd64.go:43:16: undefined: cpuid.MaxLogocalCPUId
```

--- a/go.mod
+++ b/go.mod
@@ -6,7 +6,7 @@ require (
 	github.com/9elements/go-tss v0.0.0-20200825132442-ad4c8d234e5e
 	github.com/fearful-symmetry/gomsr v0.0.1
 	github.com/google/go-tpm v0.3.1-0.20200831123702-16478836c95e
-	github.com/intel-go/cpuid v0.0.0-20181003105527-1a4a6f06a1c6
+	github.com/intel-go/cpuid v0.0.0-20200819041909-2aa72927c3e2
 	github.com/logrusorgru/aurora v0.0.0-20200102142835-e9ef32dff381
 	golang.org/x/crypto v0.0.0-20181203042331-505ab145d0a9
 	golang.org/x/sys v0.0.0-20190215142949-d0b11bdaac8a // indirect

--- a/pkg/hwapi/cpuid_amd64.go
+++ b/pkg/hwapi/cpuid_amd64.go
@@ -40,5 +40,5 @@ func (t TxtAPI) CPUSignature() uint32 {
 
 //CPULogCount returns number of logical CPU cores
 func (t TxtAPI) CPULogCount() uint32 {
-	return uint32(cpuid.MaxLogocalCPUId)
+	return uint32(cpuid.MaxLogicalCPUId)
 }

